Frameworks/SafariServices/SFSafariWebViewController.mm

```diff
@@ -177,24 +177,46 @@ - (void)loadView {
     [_webView loadRequest:request];
     [topView addSubview:_webView];
 
-    // <-
-    UIBarButtonItem* backButton =
-        [[UIBarButtonItem alloc] initWithImage:backArrow style:UIBarButtonItemStylePlain target:_webView action:@selector(goBack)];
+    StrongId<NSMutableArray<UIBarButtonItem*>> toolbarItemsArray = [[NSMutableArray alloc] init];
 
-    // ->
-    UIBarButtonItem* forwardButton =
-        [[UIBarButtonItem alloc] initWithImage:backArrow style:UIBarButtonItemStylePlain target:_webView action:@selector(goForward)];
+    UIBarButtonItem* space =
+        [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemFlexibleSpace target:nil action:nullptr];
+
+    if ([_webView canGoBack] == YES) {
+        // <-
+        UIBarButtonItem* backButton =
+            [[UIBarButtonItem alloc] initWithImage:backArrow style:UIBarButtonItemStylePlain target:_webView action:@selector(goBack)];
+        [toolbarItemsArray addObject:backButton];
+    } else {
+        // This is to ensure that the other buttons are rendered at the correct position.
+        [toolbarItemsArray addObject:space];
+    }
```

Comment:
This only works when we load the view; what happens after we navigate in the WebView?  Don't we want to update our buttons as we navigate around?